src/ert/gui/model/snapshot.py

```diff
@@ -211,9 +212,25 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
                 if job.index:
                     job_node.data[ids.INDEX] = job.index
                 if job.current_memory_usage:
-                    job_node.data[ids.CURRENT_MEMORY_USAGE] = job.current_memory_usage
+                    job_node.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
+                    real_node.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
+                    self.root.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
```

Comment:
We could avoid casting the values three times over?

```suggestion
                    current_memory_usage = float(job.current_memory_usage)

                    job_node.data[ids.CURRENT_MEMORY_USAGE] = current_memory_usage
                    real_node.data[ids.CURRENT_MEMORY_USAGE] = current_memory_usage
                    self.root.data[ids.CURRENT_MEMORY_USAGE] = current_memory_usage
```